init.go

```diff
@@ -54,35 +52,6 @@ func bzrDep(root string) (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
-func cleanDeps(vcsDeps []*godl.VCS) ([]depEntry, error) {
```

Comment:
Can we leave this stuff? This whole file is dead code, but I wanted to have it around to implement `init` or just for ideas.